_posts/en/newsletters/2020-02-12-newsletter.md

```diff
@@ -0,0 +1,111 @@
+---
+title: 'Bitcoin Optech Newsletter #84'
+permalink: /en/newsletters/2020/02/12/
+name: 2020-02-12-newsletter
+slug: 2020-02-12-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter seeks help testing a Bitcoin Core release
+candidate and summarizes some discussion about the BIP119
+`OP_CHECKTEMPLATEVERIFY` proposal.  Also included is our regular section
+about notable code and documentation changes.
+
+## Action items
+
+- **Help test Bitcoin Core 0.19.1rc1:** this upcoming maintenance
+  [release][bitcoin core 0.19.1] includes several bug fixes.
+  Experienced users are encouraged to help test for any regressions or
+  other unexpected behavior.
+
+## News
+
+- **`OP_CHECKTEMPLATEVERIFY` (CTV) Workshop:** video ([morning][ctv
+  morning vid], [afternoon][ctv afternoon vid]) and a [transcript][ctv
+  transcript] are available from a recent workshop about [BIP119][] CTV.
+  If this proposed soft fork is adopted, users would be able to use a
+  new CTV opcode to create [covenants][topic covenants] with less
+  interaction than would be required using current consensus rules.
+  Several possible applications of the opcode were discussed, with most
+  attention being focused on vaults and compressed payment batching
+  (sometimes called *congestion control transactions*, see [Newsletter #48][news48 cc]).  A significant
+  part of the workshop also consisted of critical feedback from the
+  audience and replies to the criticism.  A final discussion covered how
+  and when to attempt to get BIP119 activated, including when a PR for
+  it should be opened to the Bitcoin Core repository, what activation
+  mechanism it should use (e.g.  [BIP9][] versionbits), and what range
+  of activation dates would be appropriate if it uses a miner-activated
+  soft fork mechanism such as BIP9.
+
+    Subsequent to
+    the workshop, CTV proposer Jeremy Rubin announced a [mailing
+    list][ctv mailing list] to help coordinate future review and
+    discussion of the BIP119 proposal.
+
+## Notable code and documentation changes
+
+*Notable changes this week in [Bitcoin Core][bitcoin core repo],
+[C-Lightning][c-lightning repo], [Eclair][eclair repo], [LND][lnd repo],
+[libsecp256k1][libsecp256k1 repo], [Bitcoin Improvement Proposals
+(BIPs)][bips repo], and [Lightning BOLTs][bolts repo].*
+
+- [Bitcoin Core #17578][] FIXME:dongcarl rpc: deprecate getaddressinfo label
```

Comment:
```suggestion
- [Bitcoin Core #17585][] deprecates the `label` field returned by the
`getaddressinfo` RPC, as the `labels` (plural) field already exists and provides
the same functionality. The `label` field is expected to be removed in 0.21, for
compatibility, the old behaviour can be re-enabled in the interim by launching
`bitcoind` with `-deprecatedrpc=label`. This change is the final one in a series
of changes to clean up the `getaddressinfo` RPC interface.
```

Reply:
Looks good to me :+1: Thanks Carl!

Reply:
This LGTM too, but for completeness, I was asked in https://github.com/bitcoin/bitcoin/pull/17578#discussion_r362938617 to remove "bitcoind" after "launching" in the release notes because the deprecation argument also applies to bitcoin-qt.